examples/compute/main.c

```diff
@@ -0,0 +1,138 @@
+#ifndef WGPU_H
+#define WGPU_H
+#include "wgpu.h"
+#endif
+
+#include "framework.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define BINDINGS_LENGTH (1)
+#define BIND_GROUP_LAYOUTS_LENGTH (1)
+
+int main(
+    int argc, 
+    char *argv[]) {
+
+    if (argc != 5) {
+        printf("You must pass 4 positive integers!");
+        return 0;
+    }
+
+    uint32_t numbers[] = {
+        strtoul(argv[1], NULL, 0),
+        strtoul(argv[2], NULL, 0),
+        strtoul(argv[3], NULL, 0),
+        strtoul(argv[4], NULL, 0),
+    };
+
+    uint32_t size = sizeof(numbers);
+
+    uint32_t numbers_length = size / sizeof(uint32_t);
+
+    WGPUInstanceId instance = wgpu_create_instance();
+
+    WGPUAdapterId adapter = wgpu_instance_get_adapter(instance,
+        &(WGPUAdapterDescriptor){
+            .power_preference = WGPUPowerPreference_LowPower,
+        });
+
+    WGPUDeviceId device = wgpu_adapter_request_device(adapter,
+        &(WGPUDeviceDescriptor){
+            .extensions = NULL
+        });
+
+	uint8_t *staging_memory;
+
+    WGPUBufferId staging_buffer = wgpu_device_create_buffer_mapped(device,
```

Comment:
I don't know what to do with the formatting. In my IDE's: Visual Studio and Visual Studio code, it is perfectly formatted.
![image](https://user-images.githubusercontent.com/6235760/61994041-94770100-b075-11e9-87f6-e390f55d6ff1.png)